src/types/discord.ts

```diff
@@ -1,4 +1,9 @@
+import type { Channel, Guild } from "discord-types/general";
+import { CommandInteraction } from "../renderer/apis/commands";
+import { GetCommandOptions, RepluggedCommandResult } from "./coremods/commands";
 export enum ApplicationCommandOptionType {
+  SubCommand = 1,
```

Comment:
As the [official Discord doc](https://discord.com/developers/docs/interactions/application-commands#subcommands-and-subcommand-groups) says. Consequently rename all SubCommand to Subcommand.
```suggestion
  Subcommand = 1,
```